cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -37,6 +37,9 @@ type AutoscalingOptions struct {
 	// ScaleDownUtilizationThreshold sets threshold for nodes to be considered for scale down.
 	// Well-utilized nodes are not touched.
 	ScaleDownUtilizationThreshold float64
+	// ScaleDownGPUUtilizationThreshold sets threshold for GPU nodes to be considered for scale down.
+	// Well-utilized nodes are not touched.
+	ScaleDownGPUUtilizationThreshold float64
```

Comment:
Please make it `ScaleDownGpuUtilizationThreshold`.
Also expand comment to more explicitly say what the threshold means. (Yeah, I know comment for `ScaleDownUtilizationThresholds` is not meaningful neither; feel free to fix that one too :) ).
Specifically it is not obvious from variable name (nor comment) what resources does the utilization refer to (are those CPU, memory, or maybe GPU)

Reply:
Great! Yeah. I will make it clear.